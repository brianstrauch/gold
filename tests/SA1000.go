package main

import (
	"regexp"
	r "regexp"
)

func main() {
	const a = `(`
	const b = "("
	const c = a

	var d = `(`
	var e = "("
	var f = a

	g := `(`
	h := "("
	i := a

	// Match
	regexp.Compile("(")
	regexp.Match("(", nil)
	regexp.MatchReader("(", nil)
	regexp.MatchString("(", "")
	regexp.MustCompile("(")
	regexp.MustCompile(`(`)
	r.MustCompile("(")
	regexp.MustCompile(a)
	regexp.MustCompile(b)
	regexp.MustCompile(c)
	regexp.MustCompile(d)
	regexp.MustCompile(e)
	regexp.MustCompile(f)
	regexp.MustCompile(g)
	regexp.MustCompile(h)
	regexp.MustCompile(i)

	// No Match
	regexp.MustCompile("")
	regexp.DoNotCompile("(")
	regexp.MatchString("", "(")
	other.MustCompile("(")
}
