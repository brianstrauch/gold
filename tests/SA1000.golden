tests/SA1000.go:22:17: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:23:15: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:24:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:25:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:26:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:27:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:28:16: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:29:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:30:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:31:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:32:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:33:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:34:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:35:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:36:21: error parsing regexp: missing closing ): `(` (SA1000)
tests/SA1000.go:37:21: error parsing regexp: missing closing ): `(` (SA1000)
